web/src/beta/lib/core/StoryPanel/Block/builtin/Title/index.tsx

```diff
@@ -24,6 +24,7 @@ const TitleBlock: React.FC<Props> = ({ block, isSelected, ...props }) => {
     () => property?.title?.color?.value as ValueTypes["string"],
     [property?.title?.color?.value],
   );
+  const hasEmptySpace = /^ *$/.test(title);
```

Comment:
I don't think you need to do this. All you should need to do is change `{title ?? t("Untitled")}` to `{title || t("Untitled")}` 👍🏻

Reply:
done

Reply:
I just tested myself and it works 🤔

Reply:
https://github.com/reearth/reearth/assets/89770889/5fe37970-490e-4b0a-a9da-27f2bb10afbc

Reply:
Ah sorry I misunderstood what you meant...